frontend/src/app/modules/availability/new-slot/questions/questions.component.html

```diff
@@ -1,41 +1,59 @@
 <div class="container flex-col">
-    <mat-label class="outside-label bold">Questions for Booker</mat-label>
+    <mat-label class="outside-label title">Required Questions</mat-label>
     <!-- Mandatory questions -->
     <ng-container *ngIf="mandatoryQuestionsExist">
         <ng-container *ngFor="let question of questions; let i = index">
             <ng-container *ngIf="question.isMandatory">
                 <div class="question-mandatory list-item-wrapper flex cross-axis-center">
                     <mat-label class="question-index bold">{{ i + 1 }}.</mat-label>
+                    <mat-label class="outside-label">{{ question.questionText }}</mat-label>
+                </div>
+                <div class="question-mandatory list-item-wrapper flex cross-axis-center">
                     <mat-form-field class="question-input no-padding" appearance="outline">
-                        <input matInput type="text" [(ngModel)]="question.questionText" disabled />
+                        <input matInput type="text" [placeholder]="question.placeHolderText" disabled />
                     </mat-form-field>
+                    <div class="checkbox-container">
+                        <mat-checkbox name="questionCheckBox" checked="true"> </mat-checkbox>
+                        <span class="input-message">{{ question.checkboxText }}</span>
+                    </div>
                 </div>
             </ng-container>
         </ng-container>
         <mat-divider class="questions-divider"></mat-divider>
     </ng-container>
     <!-- Optional questions -->
-    <div cdkDropList class="optional-questions-wrapper" (cdkDropListDropped)="drop($event)">
+    <div class="flex-row">
+        <mat-label class="outside-label title">Additional Questions</mat-label>
+        <button class="plus-sign-btn" mat-flat-button (click)="addNewQuestion()">
+            <div class="flex cross-axis-center">
+                <mat-icon>add</mat-icon>
+                <span class="bold">Add New question</span>
+            </div>
+        </button>
+    </div>
+    <div cdkDropList class="optional-questions-wrapper" cdkDropListLockAxis="y" (cdkDropListDropped)="drop($event)">
```

Comment:
The _dragAndDrop_ functionality should be restricted to `optional-questions-wrapper` instead of just y-axis